python/tank/descriptor/io_descriptor/git_branch.py

```diff
@@ -123,24 +123,23 @@ def download_local(self):
             # nothing to do!
             return
 
-        # cache into the primary location
+        # cache into the temporary location
+        temporary_path = self._get_temporary_cache_path()
```

Comment:
Also, why are we not creating the parent folder here like in the appstore descriptor? Wouldn't we run the risk of not being able to move the file into the parent?

Reply:
Yes, that is possible. Will do that here as well.